test/linuxRules.cpp

```diff
@@ -252,7 +253,17 @@ TEST( Process, ProcessName )
         }
 
         child = fork();
-        ASSERT_TRUE(child >= 0);
+
+        // We are not using the GTEST ASSERT_* here as the static analysis seems to think that
+        // its possible to have a memory leak. We explicitly check the value and free memory
+        // that the analyzer is complaining about and then ASSERT.
+        if (child < 0)
+        {
+            free(cmdline);
+            cmdline = NULL;
```

Comment:
If `cmdline` is set to `NULL` here then it seems possible to leak `cmdline_copy` below.